setup.py

```diff
@@ -33,6 +33,7 @@
     cmdclass={'test': Tests},
     tests_require=[
         'mock',
+        'xmlrunner',
```

Comment:
It seems like you removed the need for `xmlrunner` when you switched to `TextTestRunner` from `unittest`. Is this still required?